src/helpers/string-to-number-validator.ts

```diff
@@ -0,0 +1,12 @@
+import { Type } from "class-transformer";
+import { IsNumber } from "class-validator";
+
+/**
+ * Checks if a value can be converted to a number
+ */
+export const StringToNumber = (): PropertyDecorator => {
```

Comment:
It would be nice with a comment here so it's a little more easy to understand what is happening :-)